data/hai/hai reveal 5 philosophers.txt

```diff
@@ -759,13 +595,69 @@ mission "Hai Reveal [B08-long] The Other Just As Fair"
 	clearance
 	passengers 1
 	to offer
-		has "Hai Reveal [B07-long] Learn of Keystones: done"
-		not "event: knows about keystones and wormholes"
+		has "Hai Reveal [B07-1] Boarded Smuggler: done"
 	on offer
 		fail "Hai Reveal [B02-S] Republic Intelligence Stalkers"
 		log `Giti tripped an alert which places you both under the jurisdiction of a Syndicate Internal Affairs investigation. You've were sent directly to Hephaestus.`
-		log "Minor People" `Agent Kopkow` `A Republic Intelligence officer who intercepted your investigation with Giti before involving you with Syndicate Internal Affairs.`
-		conversation "giti tripped an alert"
+		log "Minor People" `Agent Horst Kopkow` `A Republic Intelligence officer who intercepted your investigation with Giti before involving you with Syndicate Internal Affairs. From her unprompted knowledge of his first name, it seems this may not have been her first run-in with the law.`
+		log "Factions" "Republic Intelligence" `The top police force of the Republic, who deal with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly means it's beholden to the interests of the members of Parliament.`
+		conversation
+			`After a long series of shuttles, high-speed rails, and a several-minute-long elevator ride, you once again reach Giti's office.`
+			`	You enter the room, and she closes the door. "Captain <last>! Welcome back. Please, come in. I've been working on that translation, and I've finally figured it out. The nonsense words I thought were a riddle were actually a formula for calculating a jump target from the size of a hyperspace aperture, the origin, and a direction. I've managed the direction, and I could determine an origin if I had the aperture size."`
+			choice
+				`	"Does '0.184 steradians' sound like the answer?"`
+				`	"My favorite number has always been 0.184."`
+			`	"Hmmm, 0.184 steradians? Sounds about right; let me check." She pulls out a pencil and paper, and works out some numbers. A few minutes later, she says, "Yes, that was it. The system is in the middle of Syndicate-"`
```

Comment:
A few minutes of pencil and paper? Surely she has something more sophisticated, like protractor?